ncat/ncat_main.c

```diff
@@ -307,7 +307,7 @@ int main(int argc, char *argv[])
         {"nsock-engine",    required_argument,  NULL,         0},
         {"test",            no_argument,        NULL,         0},
         {"ssl",             no_argument,        &o.ssl,       1},
-        {"zero",            no_argument,        NULL,         'z'},
+        {"z",               no_argument,        NULL,         'z'},
```

Comment:
To add a short argument, you don't need to add to the `long_options` array, just make sure it's included in the string argument to `getopt_long` (which it is below).